java/test/behaviour/connection/ConnectionStepsBase.java

```diff
@@ -60,11 +61,7 @@ public abstract class ConnectionStepsBase {
             pair("transaction-timeout-millis", (option, val) -> option.transactionTimeoutMillis(Integer.parseInt(val)))
     );
 
-    public static final Map<String, String> serverOptions = map(
-            pair("--diagnostics.reporting.errors", "false"),
-            pair("--diagnostics.reporting.statistics", "false"),
-            pair("--diagnostics.monitoring.enable", "false")
-    );
+    public static final Map<String, String> serverOptions = Collections.emptyMap();
```

Comment:
I decided to leave it for the `Java` driver, it would be comfortable to have a single place for these additions if we will still need to update it in the future.